### ISSUE TITLE
Adjust key types based on UUID changes in qb

### ISSUE THREAD

**Issue by elpete** [coldbox-modules/quick#80]
Related to https://github.com/coldbox-modules/qb/issues/107

Adjust the key types to match the changes resulting in demystifying qb’s `uuid` column type.

**Comment by elpete** [comment]
Waiting on https://github.com/coldbox-modules/qb/issues/107

**Comment by elpete** [comment]
Addressed by PR #332, now merged into the next branch.
